pxe/PXELinux_default.erb

```diff
@@ -20,7 +20,7 @@ LABEL local
 LABEL discovery
      MENU LABEL (discovery)
      KERNEL boot/fdi-image/vmlinuz0
-     APPEND initrd=boot/fdi-image/initrd0.img rootflags=loop root=live:/fdi.iso rootfstype=auto ro rd.live.image acpi=force rd.luks=0 rd.md=0 rd.dm=0 rd.lvm=0 rd.bootif=0 rd.neednet=0 nomodeset proxy.url=https://FOREMAN_INSTANCE proxy.type=foreman
+     APPEND initrd=boot/fdi-image/initrd0.img rootflags=loop root=live:/fdi.iso rootfstype=auto ro rd.live.image acpi=force rd.luks=0 rd.md=0 rd.dm=0 rd.lvm=0 rd.bootif=0 rd.neednet=0 nomodeset proxy.url=https://<%= foreman_server_fqdn %> proxy.type=foreman
```

Comment:
Why not foreman_server_url?  It includes the protocol for the Foreman web UI, which may be HTTP.